release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3052,4 +3052,32 @@ $ oc adm release info 4.11.18 --pullspecs
 [id="ocp-4-11-18-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-20"]
+=== RHSA-2022:8893 - {product-title} 4.11.20 bug fix and security update
+
+Issued: 2022-12-15
+
+{product-title} release 4.11.20, which includes security updates, is now available. There is no IBM powerbuild for this release. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:8893[RHSA-2022:8893] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:8892[RHBA-2022:8892] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.20 --pullspecs
+----
+
+[id="ocp-4-11-20-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Google Cloud Platform used a hard-coded list to display regions in the installer survey, which resulted in several supported regions being excluded. With this update, all supported regions are included in the installer survey. (link:https://issues.redhat.com/browse/OCPBUGS-3023[*OCPBUGS-3023*])
```

Comment:
I think technically it isn't _GCP_ that had this issue, but OpenShift, specifically the installer. The installer was missing GCP regions. So this sounds like an issue in Google's GCP, but it was us.

Reply:
Yeah, I re-read the bug and realized some things were a bit off. Thanks, @jboxman-rh